Documentation delete endpoints (#293)
smartlook/smartlook.github.io: src/rest-api/cookbook.mdx

2. Get the `visitorId` from the response
3. Get `projectId` and `organizationId` from your dashboard URL
4. Now you can build the URL to the visitor detail as follows: `https://app.smartlook.com/org/{organizationId}/project/{projectId}/recordings?visitor={visitorId}`

# Deeplink to delete visitors

> 1. Check out the [API reference](/docs/rest-api/reference)

Reviewer: Tenhle krok bych klidne vynechal.

> 2. Are existing 2 endpoints for deleting visitors and their related events, sessions...

Reviewer: ```suggestion
2. There are 2 endpoints to delete visitors and their related events and sessions
```

>     1. Delete only 1 visitor by `visitorId` (unique ID obtained by Smartlook) - this `visitorId` fill in to the path

Reviewer: Tady bych pridal i tu konkretni cestu z dokumentace. Stejne tak u dalsiho bodu.

Author: Doplneno

>     2. Delete more visitors by `userId` (`UserId` is a value, which is obtained from the recorded page, where you can set this ID)

Reviewer: Vsechny `userId` by mely byt v cammelCase.

A jeste nejak upravit tu dalsi vetu, pridal bych tam odkaz na https://docs.smartlook.com/docs/web/identify-visitor?platform=web 

```suggestion
    2. Delete more visitors by `userId` (`userId` is set by identify method in the SDK)
```
